src/mail/view/MailViewer.ts

```diff
@@ -678,9 +679,38 @@ export class MailViewer implements Component<MailViewerAttrs> {
 			})
 	}
 
+	private async getAvailableFolders(): Promise<MailFolder[]> {
+		const folders = await this.viewModel.mailModel.getMailboxFolders(this.viewModel.mail)
+		const filteredFolders = folders.filter(f => f.mails !== this.viewModel.getMailId()[0])
```

Comment:
should be `listIdPart(this.viewModel.getMailId())`

Reply:
Must have been from the code I extracted. I went and fixed this.